README.md

```diff
@@ -118,14 +118,28 @@ Compiling
 
 ### Compiling on GNU/Linux
 
-Install dependencies. Here's an example for Debian/Ubuntu:
+#### Dependencies
+
+| Dependency | Version | Commentary |
+|------------|---------|------------|
+| GCC        | 4.9+    | Can be replaced with Clang 3.4+ |
+| CMake      | 2.6+    |            |
+| Irrlicht   | 1.7.3+  |            |
+| SQLite3    | 3.0+    |            |
+| LuaJIT     | 2.0+    | Bundled Lua is used if not present |
```

Comment:
here you could mention that bundeled lua is 5.1

Reply:
Very carefulness :)